applications/luci-app-xfrpc/htdocs/luci-static/resources/view/xfrpc.js

```diff
@@ -0,0 +1,176 @@
+		s.tab('init', _('Startup Settings'));
+
+		o = s.taboption('common', form.Value, 'server_addr', _('Server address'), _('Server address specifies the address of the server to connect to.<br />By default, this value is "0.0.0.0".'));
+		o = s.taboption('common', form.Value, 'server_port', _('Server port'), _('Server port specifies the port to connect to the server on.<br />By default, this value is 7000.'));
+		o = s.taboption('common', form.Value, 'token', _('Token'), _('Token specifies the authorization token used to create keys to be sent to the server. The server must have a matching token for authorization to succeed. <br />By default, this value is "".'));
+
+		o = s.taboption('init', form.Flag, 'disabled', _('Disabled xfrpc service'));
+		o = s.taboption('init', form.ListValue, 'loglevel', _('Log level'), _('LogLevel specifies the minimum log level. Valid values are "debug", "info", "notice", "warning", "err", "crit", "alert" and "emerg".<br />By default, this value is "info".'));	
+		o.value('7');
+		o.value('6');
+		o.value('5');
+		o.value('4');
+		o.value('3');
+		o.value('2');
+		o.value('1');
+		o.value('0');
+
+		s = m.section(form.GridSection, 'xfrpc', _('Proxy Settings'));
+		s.addremove = true;
+		s.filter = function(s) { return s !== 'common'; };
+		s.renderSectionAdd = function(extra_class) {
+			var el = form.GridSection.prototype.renderSectionAdd.apply(this, arguments),
+				nameEl = el.querySelector('.cbi-section-create-name');
+			ui.addValidator(nameEl, 'uciname', true, function(v) {
+				if (v === 'common') return _('Name can not be "common"');
+				return true;
+			}, 'blur', 'keyup');
+			return el;
+		}
+
+		s.tab('general', _('General Settings'));
+		s.tab('http', _('HTTP Settings'));
+
+		s.option(form.Value, 'type', _('Proxy type')).modalonly = false;
+		s.option(form.Value, 'local_ip', _('Local IP')).modalonly = false;
+		s.option(form.Value, 'local_port', _('Local port')).modalonly = false;
+
+		o = s.taboption('general', form.ListValue, 'type', _('Proxy type'), _('ProxyType specifies the type of this proxy. Valid values include "tcp", "http", "https".<br />By default, this value is "tcp".'));
+		o.value('tcp');
+		o.value('http');
+		o.value('https');
+		o.modalonly = true;
+		o = s.taboption('general', form.Value, 'local_ip', _('Local IP'),  _('LocalIp specifies the IP address or host name to proxy to.'));
+		o.modalonly = true;
+		o = s.taboption('general', form.Value, 'local_port', _('Local port'), _('LocalPort specifies the port to proxy to.'));
+		o.modalonly = true;
+
+		// TCP
+		o = s.taboption('general', form.Value, 'remote_port', _('Remote port'), _('If remote_port is 0, frps will assign a random port for you'));
+		o.depends.apply(o, [{type: 'tcp'}]);
+		o.optional 	= true;
+		o.modalonly = true;
+
+		// HTTP and HTTPS
+		o = s.taboption('http', form.Value, 'custom_domains', _('Custom domains'));
+		o.depends.apply(o, [{type: 'http'}]);
+		o.depends.apply(o, [{type: 'https'}]);
+		o.optional 	= true;
```

Comment:
Either all `=` under each other or a blank before and after the `=`, but please do not mix them.
Please always consistent. I prefer this with the space character